### DISCUSSION TITLE
Any plans to support gzipped files?

### DISCUSSION THREAD

**Discussion by prerak-proof** [mithrandie/csvq#73] [Ideas]
Many of our historical csv files are gzipped and it would be cool to be able to query them without first unzipping.

**Comment by mithrandie** [comment]
I will support it in the future. I plan to support gzip first, but if you have any requests for other formats, please post them in this issue.

https://github.com/mithrandie/csvq/issues/81

**Comment by ondohotola** [comment]
do a pipeline and select from stdin

el

—
